sportinggoods.js

```diff
@@ -0,0 +1,48 @@
+//List of products
+
+const bat = {
```

Comment:
great job converting these into object literals and making an inventory array. Looks great!